Gemfile.lock

```diff
@@ -444,4 +454,4 @@ RUBY VERSION
    ruby 2.5.1p57
 
 BUNDLED WITH
-   1.16.1
+   1.16.4
```

Comment:
As explained in my comments above: as it's not really relevant for the actual changes in this PR, it would be best if you could just unstage the changes on the `Gemfile` and `Gemfile.lock`. You can still keep these changes on your local fork if they make working with the app easier for you - just exclude them from this PR ✌️